config/config.go

```diff
@@ -37,6 +38,10 @@ const (
 	DefaultCommittedLocalCacheMetaRangePercent = 0.1
 	DefaultCommittedLocalCacheBytes            = 1 * 1024 * 1024 * 1024
 	DefaultCommittedLocalCacheDir              = "~/lakefs/local_tier"
```

Comment:
Can you please make sure we're not somehow getting rid of the tilde-expansion fix that I added?